src/components/CommunityBanner.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+interface Props {
+  className?: string;
+  title?: string;
+  text?: string;
+  bg?: "bg_community_1" | "bg_community_2" | "bg_community_3";
+}
+
+const CommunityBanner: React.FC<Props> = ({ className, text, title, bg }) => {
+  const bgMapping = {
+    bg_community_1: "lg:bg-community_multi_1_lg bg-community_multi_1_sm",
```

Comment:
How about turning this `bgMapping` into an array, so the `bg` prop is just a number between 1 and 3. Don't need these long object keys (they make sense when you have different icons, but not when it's just numeric values).

Reply:
done! i'll commit this change alone! since it's not a fix, it'll call the commit a feat: update community banner component. hope it makes sense!